iam.tf

```diff
@@ -12,22 +12,28 @@ data "aws_iam_policy_document" "assume_role" {
   }
 }
 
+locals {
+  eks_service_role = var.eks_cluster_service_role_arn != null ? var.eks_cluster_service_role_arn : join("", aws_iam_role.default.*.arn)
+
+  create_eks_service_role = var.eks_cluster_service_role_arn == null && local.enabled
```

Comment:
I think it's nicer to reorder it so that `local.enabled` comes first — it's more visible when the boolean variable is in the same locals block where `enabled` is. But this doesn't matter. Just what I've seen.

Reply:
For example see https://github.com/cloudposse/terraform-aws-eks-cluster/blob/master/main.tf#L6

Reply:
And if you do make changes, I would also take the opportunity to make sure all locals reference `local.enabled` instead of `module.this.enabled` https://github.com/cloudposse/terraform-aws-eks-cluster/blob/master/main.tf#L9

Reply:
Done